tizen/src/tasks.h

```diff
@@ -0,0 +1,111 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_TASKS_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_TASKS_H_
+
+#include <optional>
+#include <string>
+#include <variant>
+
+#include "options.h"
+
+class WorkManagerCall {
+   public:
+    WorkManagerCall(){};
+};
```

Comment:
Is there a reason to declare this class and why other classes should inherit this class?